metadata-ingestion/src/datahub/ingestion/source/file.py

```diff
@@ -406,22 +409,25 @@ def _from_obj_for_file(
     if not item.validate():
         raise ValueError(f"failed to parse: {obj}")
 
-    return item
+    if isinstance(item, UsageAggregationClass):
+        logger.warning(f"Dropping deprecated UsageAggregationClass: {item}")
```

Comment:
let's add a todo here - in the future, we'll want to change this so that it throws an error instead

Reply:
Will we not just entirely remove the class? I figured todos were not necessary and if you remove the class then you are forced to remove all references to the class